Frontend/items.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE HTML>
+<html lang='en'>
+<head>
+    <title>Welcome to Lean Fridge</title>
+
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css' integrity='sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7' crossorigin='anonymous'>
+
+<script type = 'text/javascript'>
+    var stock_array = [
```

Comment:
name this just `stock`. _array is in general superflous (unless you need to distinguish it from something else)